cmd/ak/cmd/deployments/delete.go

```diff
@@ -0,0 +1,40 @@
+package deployments
+
+import (
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var deleteCmd = common.StandardCommand(&cobra.Command{
+	Use:     "delete <deployment ID> [--fail]",
+	Short:   "Delete inactive deployment",
+	Aliases: []string{"d"},
+	Args:    cobra.ExactArgs(1),
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		r := resolver.Resolver{Client: common.Client()}
+		d, id, err := r.DeploymentID(args[0])
+		if err != nil {
+			return common.FailIfError(cmd, err, "deployment")
+		}
+
+		if err := common.FailIfNotFound(cmd, "deployment", d); err != nil { // test d != nil
+			return err
+		}
```

Comment:
do we really need to check if deployments exists
and then try to delete it ?
we will get the same result by just calling delete if it exists it will be deleted (assuming it can reach inactive state)
if it is not found, we will get not found error

Reply:
I'm following the logic of other cmd commands. If you think it's not needed, let's discuss and open a ticket to fix all commands?